src/Tizen.Multimedia/MediaTool/MediaFormatVideoMimeType.cs

```diff
@@ -37,6 +37,60 @@ public enum MediaFormatVideoMimeType
         /// </summary>
         H263P = (MediaFormatType.Video | MediaFormatDataType.Encoded | 0x2021),
 
+        /// <summary>
+        /// H263 Baseline Profile.
+        /// </summary>
+        /// <since_tizen> 13 </since_tizen>
+        H263BLP = (MediaFormatType.Video | MediaFormatDataType.Encoded | 0x2022),
+
+        /// <summary>
+        /// H263 H.320 Coding Efficiency Profile.
+        /// </summary>
+        /// <since_tizen> 13 </since_tizen>
+        H263H320CEP = (MediaFormatType.Video | MediaFormatDataType.Encoded | 0x2023),
+
+        /// <summary>
+        /// H263 Backward-Compatibility Profile.
+        /// </summary>
+        /// <since_tizen> 13 </since_tizen>
+        H263BCP = (MediaFormatType.Video | MediaFormatDataType.Encoded | 0x2024),
+
+        /// <summary>
+        /// H263 the version 2 Interactive and Streaming Wireless Profile.
```

Comment:
I think it would sound better without "the", just "H263 version 2, Interactive ..."
same with the H263ISW3P

Reply:
I fixed it.